lsdcGui.py

```diff
@@ -2280,13 +2280,16 @@ def createSampleTab(self):
         self.zoomRadioGroup.addButton(self.zoom1Radio)
         self.zoom2Radio = QtWidgets.QRadioButton("Mag2")
         self.zoom2Radio.toggled.connect(functools.partial(self.zoomLevelToggledCB,"Zoom2"))
-        self.zoomRadioGroup.addButton(self.zoom2Radio)
+        if (daq_utils.sampleCameraCount>=2):
```

Comment:
Do we want the option to have something like Mag1, Mag3, Mag4? My assumption was that it would always be a sequence.

Reply:
I don't think so, but mentioning that sampleCameraCount implies a sequential camera numbering is good.

Reply:
the way the fix has been done leads to other problems - if you check the code for references to self.zoom3Radio, you can see that it is used at, for example, in pixelSelect - line 4267. if you only have two zoom levels, it will actually hit the `if self.zoom4Radio` and throw an exception because that is not defined.

I would suggest defining all of the buttons and add the toggled callback in the main code area, but add the buttons to self.zoomRadioGroup in the `if >=2...` block